src/development/coding-licences.md

```diff
@@ -20,6 +20,47 @@ This should be called `LICENCE` or `LICENCE.md`. “License” is the U.S. Engli
 
 Make sure the licence content is included in full, including the title “APACHE LICENSE, VERSION 2.0”, so that readers are quickly able to see which licence is being used.
 
+### Forked projects
+
+If you have forked a project that was licensed under Apache 2 the following additional changes are required:
```

Comment:
I think we could add a bit more about the need to comply with licence obligations. And it'd be great to have a link to some external guidance for all licences if possible. I used to have one, but I've lost ot